database/frontend/icons.json

```
@@ -220,7 +220,14 @@
     "name": "Iconoir",
     "description": "A high-quality selection of free icons. Has over 1,368 icons ready to use for your projects.",
     "url": "https://iconoir.com/",
-    "category": "fontend",
+    "category": "frontend",
+    "subcategory": "icons"
+  },
+  {
+    "name": "Lucide",
+    "description": "Lucide is a Beautiful & consistent icon toolkit made by the community.",
```

[proofreading] Interesting resource! :)  However, when you're describing your resources, avoid doing things like capitalizing words that are not at the beginning of a sentence (programming languages are an exception to this). It makes descriptions unclear.  
```suggestion
    "description": "Lucide is a beautiful & consistent icon toolkit made by the community.",
```